src/App.tsx

```diff
@@ -1,6 +1,8 @@
-import React from 'react';
+import React, { useState } from 'react';
```

Comment:
nit: Import useState directly `import { useState } from 'react';`. No need to import React since you are not using it directly 🧐